config/initializers/filter_parameter_logging.rb

```diff
@@ -1,4 +1,8 @@
 # Be sure to restart your server when you modify this file.
 
-# Configure sensitive parameters which will be filtered from the log file.
-Rails.application.config.filter_parameters += [:password, :private_key, :public_key, :otp_attempt]
```

Comment:
This feature uses partial regex matching, so the below patterns cover everything here.